### PR TITLE
feat:Versioning of the APIs of the Discovery Finder

### PR DESCRIPTION
**Pull request by shijinrajbosch**
## Description

Introduced versioning of the APIs of the Discovery Finder

- Introduced versioning of the APIs of the Discovery Finder
- First version of this API is 1.0.

### CODE REVIEW

**Comment by agg3fe** [comment]
@shijinrajbosch  I think you need to add new version entry in Changelog.md file also.
right @tunacicek ? and then we can merge and create new image for this.

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres
 to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.2.5-M1
```
Comment (agg3fe) [inline CHANGELOG.md:8]: please make the version as 0.2.5 only and remove '-M1' from it.

Comment (shijinrajbosch) [reply CHANGELOG.md]: Thanks for the review comment @agg3fe  . I have incorporated the review comment.

**Review by tunacicek — APPROVED** [review]
LGTM

**Pull request merged**